src/sds.ts

```diff
@@ -794,7 +794,7 @@ export class SDSConnection {
         });
     }
 
-    public pdcCallOperation(operation: string, parameters = [], debug = false): Promise<string[]> {
+    public pdcCallOperation(operation: string, parameters: string[], debug = false): Promise<string[]> {
```

Comment:
Tow things:

1. Here you named the first argument `operation`. In `Message.pdcCallOperation` you named it `className` (which is correct, I believe). Why did you name differently here?
2. I would not use abbreviations that only people can understand that have actually looked at the server's source code. I would not name this `pdcCallOperation`.  Maybe `classCallOperation` or `callOperationClass` is better?